packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -40,12 +51,14 @@ function UnforwardedToolbar(
 		className
 	);
 	return (
-		<ToolbarContainer
-			className={ finalClassName }
-			label={ label }
-			ref={ ref }
-			{ ...props }
-		/>
+		<ContextSystemProvider value={ CONTEXT_SYSTEM_VALUE }>
```

Comment:
This won't have any effect on dropdowns inside `Toolbar` just yet, since it's still using the legacy `DropdownMenu` component. But it's setting the ground.